commands.yaml

```diff
@@ -18,21 +18,21 @@
   check_command: |
     defaults read /Library/Preferences/com.apple.Bluetooth ControllerPowerState | grep 0
   fix_command: |
-    sudo defaults write /Library/Preferences/com.apple.Bluetooth ControllerPowerState -int 0 &&  sudo killall -HUP blued
+    sudo defaults write /Library/Preferences/com.apple.Bluetooth ControllerPowerState -int 0; sudo killall -HUP blued
   enabled: true
 
 
 - title: "Disable infrared receiver"
   check_command: |
     defaults read /Library/Preferences/com.apple.driver.AppleIRController DeviceEnabled | grep 0
   fix_command: |
-    defaults write /Library/Preferences/com.apple.driver.AppleIRController DeviceEnabled -int 0
+    sudo defaults write /Library/Preferences/com.apple.driver.AppleIRController DeviceEnabled -int 0
   enabled: true
 
 
 - title: "Disable AirDrop"
   check_command: |
-    defaults read com.apple.NetworkBrowser DisableAirDrop | grep 1
+    sudo defaults read com.apple.NetworkBrowser DisableAirDrop | grep 1
```

Comment:
I agree, checking root one time at the beginning make more sens. But I see you merged that just after you wrote your comment.